src/test/java/address/testutil/ContextMenuChoice.java

```diff
@@ -0,0 +1,8 @@
+package address.testutil;
+
+/**
+ *
+ */
+public enum ContextMenuChoice {
+        EDIT, TAG, DELETE, CANCEL;
+}
```

Comment:
Can be define this in another class? Doesn't look like it should be on its own, at least based on current class name.